src/main/java/org/breedinginsight/model/ProgramUpload.java

```diff
@@ -30,25 +31,27 @@
 import org.breedinginsight.dao.db.tables.pojos.BatchUploadEntity;
 import org.jooq.Record;
 
+import java.util.List;
+
 import static org.breedinginsight.dao.db.Tables.BATCH_UPLOAD;
 
 @Getter
 @Setter
 @Accessors(chain=true)
 @ToString
-@SuperBuilder
```

Comment:
Guessing this was removed because there was an ambiguous "builder()" method?  If so, the name of the builder for this class can be changed by updating the `@Superbuilder` annotation to something like: `@SuperBuilder(builderMethodName = "uploadBuilder")`

The use of the builder will need to be typed given that `ProgramUpload` now has generic support for the `parsedData` field.  An example of how to do this would be: `ProgramUpload.<T>uploadBuilder()...`

Reply:
Perfect! Yeah the type ProgramUpload was messing things up and not liking `ProgramUpload.<Trait>builder()...`. I'll try out the named builder and see how that goes.

Reply:
Changed and pushed